docs/platforms/apple/common/index.mdx

```diff
@@ -1,34 +1,172 @@
-<PlatformContent includePath="getting-started-primer" />
+On this page, we get you up and running with Sentry's Apple SDK, automatically reporting errors and exceptions in your application.
 
-On this page, we get you up and running with Sentry's SDK.
+<Note>
+
+If you don't already have an account and Sentry project established, head over to [sentry.io](https://sentry.io/signup/), then return to this page.
+
+</Note>
+
+<PlatformSection supported={["apple.visionos"]}>
+
+<Alert level="info" title="Experimental Support">
 
-Don't already have an account and Sentry project established? Head over to [sentry.io](https://sentry.io/signup/), then return to this page.
+The support for [visionOS](https://developer.apple.com/visionos/) is currently experimental.
+Experimental support means it may have bugs. We recognize the irony.
+
+</Alert>
+
+</PlatformSection>
```

Comment:
```suggestion
Sentry captures data by using an SDK within your application's runtime. These are platform-specific and allow Sentry to have a deep understanding of how your application works.

</Alert>

</PlatformSection>
```

Reply:
is this intentional? I feel it still makes sense to show this for all platforms and outside of the alert